src/mlpack/core/optimizers/cmaes/test_function.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file test_function.hpp
```

Comment:
actually your right. First i created a different function due to parameter changes and later made it according to other functions only.

Done